packages/cli/package.json

```diff
@@ -19,7 +19,7 @@
     "/oclif.manifest.json"
   ],
   "engines": {
-    "node": ">=18"
+    "node": ">=18.18"
```

Comment:
Note this stricter requirement; eslint 9 requires the [18.18 minor version](https://nodejs.org/en/blog/release/v18.18.0), so I figure it makes sense to enforce this for at least `zapier-platform-cli`